generate_and_apply_patch.sh

```diff
@@ -6,6 +6,23 @@ BOOT=${1:-/boot}
 IRFS_HOOK="/etc/initramfs-tools/hooks/acpi_override.sh"
 cd `mktemp -d`
 
+if [ "$(iasl -v | grep -c 20180811)" -eq 0 ]; then
+ #Build a newer version of iasl
+ if [ -x $(which apt-get) ]; then
+     echo "[*] Installing required tools to build iasl"
+     sudo apt-get -y install bison flex
+ fi
+ echo "[*] Upgrading IASL..."
+ wget https://acpica.org/sites/acpica/files/acpica-unix2-20180810.tar.gz
```

Comment:
personally I'd be happier if we validated the sha sum of this download....

Reply:
for the record it only worked for me with https://acpica.org/sites/acpica/files/acpica-unix2-20181031.tar.gz

also I had to run `sudo apt install bison flex`